lib/fs.js

```diff
@@ -43,6 +43,18 @@ const internalUtil = require('internal/util');
 const assertEncoding = internalFS.assertEncoding;
 const stringToFlags = internalFS.stringToFlags;
 const getPathFromURL = internalURL.getPathFromURL;
+const O_RDONLY = constants.O_RDONLY | 0;
```

Comment:
Right, I wasn't sure of the upper ceiling for SMIs offhand, but yes that's probably what it would be.